### PR TITLE
make diagram initialization more robust

### LINKED ISSUE #584: Reloading of visu fails
If you select a different page, the url gets appended by e.g. #id_6_. If you reload the whole visu using the browser reload button, the visu won't load again. Reproduced with Safari and Chrome.

### PR DESCRIPTION
**Pull request by peuter**
closes #584

### CODE REVIEW

File: source/class/cv/ui/PageHandler.js [hunk 1]
```diff
@@ -84,8 +84,6 @@ qx.Class.define('cv.ui.PageHandler', {
 
       templateEngine.currentPage = pageWidget;
 
-      qx.event.message.Bus.dispatchByName("path."+target+".duringPageChange", target);
```
Comment (ChristianMayer) [inline source/class/cv/ui/PageHandler.js]: Isn't this a more generic infrastructure to allow widgets and plugins do preperation work to show as soon as the page is available?
=> I guess it should still be in the code

Comment (peuter) [inline source/class/cv/ui/PageHandler.js]: I deleted is for two reasons:

1. No one is using it
2. There is no "during" page change. Because Javascript is single-threaded you can only do something before the page changes or after it changed. If you count in animations we might find a use case for for doing something when the page gets visible (and the animation starts). You can already listen to an "changeVisible" event on a page object to capture that moment. 
But it might be arguable to introduce a new bus message for this, when there is a need for it.

So long story short. As soon as there is a need for it I'm open to adding another bus event, but currently its not needed.
